src/simulation/m_global_parameters.fpp

```diff
@@ -525,6 +534,10 @@ contains
             teno = .false.
         #:endif
 
+        chem_params%advection = .true.
```

Comment:
shouldn't these all default to false so user has to activate them? also we don't usually put derived type assignments here

Reply:
Sure, I don't mind these being false by default. Though, it could be confusing to new users that don't read the docs.

We usually don't have derived types that define "simulation algorithm parameters" but I think it's a good idea to modularize them like this. I might rename these in the future. `chem_params%advection` is readable and explicit in a case file but `chemistry%advection` would be even better. It's just that the `chemistry` input is a logical kill-switch right now.

Reply:
I see this as an implementation detail of chemistry that will be resolved in the full PR. Let me know if you disagree.